src/env.rs

```diff
@@ -35,8 +36,11 @@ use crate::Data;
 /// example of the latter is setting a value for enabled/disabled status
 /// so that an entire subtree can be disabled ("grayed out") with one
 /// setting.
-#[derive(Clone, Default)]
-pub struct Env(Arc<EnvImpl>);
+#[derive(Clone)]
+pub struct Env {
+    items: Arc<EnvImpl>,
+    l10n: Arc<L10nManager>,
```

Comment:
I think I'd rather this goes into EnvImpl rather than Env because it's an additional pointer. But probably not that big a deal in the grand scheme of things.

Reply:
This is tricky, because `L10nManager` can't impl `Clone`, which isn't impl'd for `FluentBundle`; this is the main reason it's behind the `Arc`.

Reply:
Understood. To be clear, I'm not talking about getting rid of the `Arc`. Because of the extremely large amount of sharing that's expected, I expect multiple levels of `Arc` and tried to design it so that it's a single pointer. If it's two pointers, it's still ok, but if it becomes a lot of state, it could be a problem.